test/0.4.24/helpers/publicKeyArrays.js

```diff
@@ -0,0 +1,14 @@
+const { hexConcat, padKey, padSig } = require('../../helpers/utils')
+
+const createKeys = (numKeys) => Array.from({ length: numKeys }, (_value, i) => padKey(`0x${i + 1}`))
```

Comment:
seems to be unused?

Reply:
Fair enough, this can wait until the full PR as we're going to be generating much more keys in there.